File: solutions/beecrowd/1099/1099.py
n = int(input())


for _ in range(n):
    x, y = map(int, input().split())
    x, y = min(x, y), max(x, y)

    x += 1
    if x % 2 == 0:
        x += 1

    s = 0
    for odd_number in range(x, y, 2):
        s += odd_number
    print(s)

File: solutions/beecrowd/1099/1099.clj
(ns main
  (:require [clojure.string :as str]))

(defn main []
  (let [n (Integer/parseInt (read-line))]
    (loop [index 0]
      (when (< index n)
        (let [[x y] (map #(Integer/parseInt %) (str/split (read-line) #" "))
              [x y] [(inc (min x y)) (max x y)]]
          (if (zero? (mod x 2))
            (println (apply + (range (inc x) y 2)))
            (println (apply + (range x y 2)))))
        (recur (inc index))))))

(main)
